[docs]: Contributing Guide Mac OS Instructions - Need to update npm version that is included with Node 14.17.3

### Summary

I was following the Mac OS instructions and came across an issues using the npm version (v6.14.13) that is included with node 14.17.3 that you switch to using nvm.

One issue When running the npm install commands in the documentation with that version of npm, you get the warning `This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!` and it then converts the lockfile back from version 2 to version 1.

The other issue is that you can't build the plugins. When running `npm build:plugins` I got the output: 

`npm ERR! code ENOENT`
`npm ERR! syscall access`
`npm ERR! path /Users/[username]/repos/ToolJet/plugins/node_modules/@tooljet-plugins/airtable`
`npm ERR! errno -2`
`npm ERR! enoent ENOENT: no such file or directory, access` `'/Users/[username]/repos/ToolJet/plugins/node_modules/@tooljet-plugins/airtable'`
`npm ERR! enoent This is related to npm not being able to find a file.`
`npm ERR! enoent`
`npm ERR! code ELIFECYCLE`
`npm ERR! errno 254`
`npm ERR! @ prebuild:plugins: 'npm --prefix plugins install'`
`npm ERR! Exit status 254`
`npm ERR!`
`npm ERR! Failed at the @ prebuild:plugins script.`
`npm ERR! This is probably not a problem with npm. There is likely additional logging output above.`

I first tried using `nvm install --latest-npm` but got the error that the npm version 7.20.0 was expected.

To alleviate this, I updated the npm version within the nvm install of node version 14.17.3.

`cd ~/.nvm/versions/node/v14.17.3/lib`
`npm install npm@7.20.0`

After this I was able to install and successfully build the plugins.



### Issue Type

Documentation bug

### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/contributing-guide/setup/Mac%20OS

### Steps to reproduce the issue

1. Install Homebrew (via the docs)
`/bin/bash -c "(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"`

2. Install and setup nvm (via the docs)
`brew install nvm
export NVM_DIR=~/.nvm
source $(brew --prefix nvm)/nvm.sh
nvm install 14.17.3
nvm use 14.17.3`

3. Install dependencies and Build the plugins (via the docs)
`npm install`
`npm install --prefix server`
`npm install --prefix frontend`
`npm run build:plugins`

### Additional Information

If you run the snippet below **after** installing _and_ selecting node version 14.17.3 with nvm and **before** installing the dependencies and building the plugins via npm, you can avoid these errors.

`cd ~/.nvm/versions/node/v14.17.3/lib`
`npm install npm@7.20.0`

### If the issue is confirmed, would you be willing to submit a pull request?

Yes

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

--- a/docs/docs/contributing-guide/setup/Mac OS.md
+++ b/docs/docs/contributing-guide/setup/Mac OS.md
@@ -20,8 +20,7 @@ Follow these steps to setup and run ToolJet on macOS for development purposes. O
     source $(brew --prefix nvm)/nvm.sh
     nvm install 14.17.3
     nvm use 14.17.3
-    cd ~/.nvm/versions/node/v14.17.3/lib
-    npm install npm@7.20.0
+    npm install -g npm@7.20.0
     ```
 
     1.3 Install Postgres

--- a/docs/docs/contributing-guide/setup/Mac OS.md
+++ b/docs/docs/contributing-guide/setup/Mac OS.md
@@ -13,13 +13,15 @@ Follow these steps to setup and run ToolJet on macOS for development purposes. O
     ```bash
     /bin/bash -c "(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
     ```
-    1.2 Install Node.js ( version: v14.17.3 )
+    1.2 Install Node.js ( version: v14.17.3 ) and npm (version: v7.20.0)
     ```bash
     brew install nvm
     export NVM_DIR=~/.nvm
     source $(brew --prefix nvm)/nvm.sh
     nvm install 14.17.3
     nvm use 14.17.3
+    cd ~/.nvm/versions/node/v14.17.3/lib
+    npm install npm@7.20.0
     ```
 
     1.3 Install Postgres

